### PR TITLE
Update MAMP Documentation regarding settings to connect #changed

### PR DESCRIPTION
**Pull request by Braveatom**
## Changes: Adding clarity to the settings change needed with MAMP Pro to make editing with Sequel Ace work.
- 

## Closes following issues:
- Closes: #1487

## Tested:
- Processors:
  - [ ] Intel
  - [ ] Apple Silicon
- macOS Versions:
  - [ ] 10.12.x (Sierra)
  - [ ] 10.13.x (High Sierra)
  - [ ] 10.14.x (Mojave)
  - [ ] 10.15.x (Catalina)
  - [ ] 11.x (Big Sur)
  - [ ] 12.x (Monterey)
- Localizations:
  - [x] English
  - [ ] Spanish
  - [ ] Other (please specify)
- Xcode Version:

### CODE REVIEW

File: docs/get-started/mamp-xampp.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ You can also connect via a TCP/IP connection.
 
 Enter 127.0.0.1 for the Host. Enter root for the username and for the password. The default MySQL port used by MAMP is 8889.
 
-**Important**: Under Server & Services -> MySQL, check `Allow network access to MySQL` and select `only from this Mac`.
+**Important**: Regardless of the connection method, a preference needs changing in the MAMP Pro settings. Under Server & Services -> MySQL, check `Allow network access to MySQL` and select `only from this Mac`.
```
Comment (Jason-Morcos) [inline docs/get-started/mamp-xampp.md:34]: This looks great! Slightly tweaked:
```
Regardless of the connection method being used, a preference must be changed in the MAMP Pro settings in order to connect via Sequel Ace.
```

Comment (Braveatom) [reply docs/get-started/mamp-xampp.md:34]: We did it! Thanks Jason :)

Comment (Kaspik) [reply docs/get-started/mamp-xampp.md:34]: Planning to update this PR? :)

**Comment by Jason-Morcos** [comment]
Merging and editing manually

**Pull request merged**